src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,32 +1,76 @@
 package edu.kis.vh.nursery;
 
+/**
+ * @author user
+ *
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
-	public int total = -1;
+	/**
+	 * Value to return when there is no numbers
+	 */
+	private static final int DEFAULT_RETURN_VALUE = -1;
+	/**
+	 * Maximum amount of numbers to store
+	 */
+	private static final int STACK_SIZE = 12;
 
+	/**
+	 * Numbers stored in rhymer
+	 */
+	private int[] NUMBERS = new int[STACK_SIZE];
+	/**
+	 * Amount of stored numbers
+	 */
+	private int total = DEFAULT_RETURN_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)